buildingmotif/dataclasses/library.py

```diff
@@ -460,6 +498,14 @@ def name(self, new_name: str):
         self._bm.table_connection.update_db_library_name(self._id, new_name)
         self._name = new_name
 
+    @property
+    def graph_imports(self) -> List[str]:
+        """
+        Get the list of owl:imports for this library's shape collection
+        """
+        shape_col = self.get_shape_collection()
+        return list(map(str, shape_col.graph.objects(None, rdflib.OWL.imports)))
```

Comment:
Don't we lose information by casting to a `str`? should we just keep the native representation and leave it to the user to cast it to str if needed?

Reply:
I've changed this so it returns the import URIs, and then converts to string elsewhere (when loading the library by name)